index.html

```diff
@@ -10,16 +10,17 @@
     <script type="systemjs-importmap">
       {
         "imports": {
-          "single-spa": "https://unpkg.com/single-spa@4.3.3/lib/system/single-spa.min.js",
+          "single-spa": "https://unpkg.com/single-spa@4.3.5/lib/system/single-spa.min.js",
           "single-spa-playground": "/assets/single-spa-playground.js"
         }
       }
     </script>
-    <script src="https://unpkg.com/import-map-overrides@1.6.1/dist/import-map-overrides.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/systemjs/4.0.1/system.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/systemjs/4.0.1/extras/amd.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/systemjs/4.0.1/extras/named-exports.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/systemjs/4.0.1/extras/named-register.min.js"></script>
+    <script src="https://unpkg.com/import-map-overrides@1.7.2/dist/import-map-overrides.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/systemjs/4.1.0/system.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/systemjs/4.1.0/extras/amd.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/systemjs/4.1.0/extras/named-exports.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/systemjs/4.1.0/extras/named-register.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/systemjs/4.1.0/extras/use-default.min.js"></script>
```

Comment:
See https://github.com/systemjs/systemjs/releases/tag/v4.1.0 for why we need this